examples/Receiver_Response.py

```diff
@@ -105,6 +109,8 @@ def get_freqs(thisfile):
 # }}}
 # }}}
 # {{{ Calculate dg
+# Get nodenames and frequencies recorded on receiver
+rec_nodenames, rec_frqs = get_freqs(file2)
```

Comment:
this was moved down for organization so that the stuff that accommodate file2 are where we use file2